popmon/visualization/alert_section_generator.py

```diff
@@ -152,9 +152,8 @@ def transform(
             features_w_metrics.append(
                 {
                     "name": feature,
-                    "types": ["traffic_lights"],
+                    "plot_type_layouts": zip(["traffic_lights"],[""]),
```

Comment:
When the contents is static, zip is not really needed.

A list with a single tuple would have the same effect:
`[("traffic_lights", "")]`